pegasus/routes/v2_section_routes.rb

```diff
@@ -13,12 +14,20 @@
   redirect "/v2/sections/#{section_id}"#, 201 #BUGBUG: JQuery is barfing on the 201
 end
 
+# Get the set of sections that the current user is enrolled in.
+get '/v2/sections/membership' do
+  only_for 'code.org'
+  dont_cache
+  content_type :json
+  JSON.pretty_generate(DashboardSection.fetch_student_sections(dashboard_user_id))
+end
+
 get '/v2/sections/:id' do |id|
   only_for 'code.org'
   dont_cache
-  forbidden! unless section = DashboardSection.fetch_if_allowed(id, dashboard_user_id)
+  forbidden! unless section = DashboardSection.fetch_if_teacher(id, dashboard_user_id)
```

Comment:
great, thanks. If we end up needing this endpoint for we can switch on student-ness/teacher-ness to call one of the filtered-hash methods